pwiz_tools/Skyline/app.config

```diff
@@ -11,817 +11,817 @@
       </sectionGroup>
     </configSections>
     <userSettings>
-        <pwiz.Skyline.Properties.Settings>
```

Comment:
Too much change. Need to go back to the original and apply only the appropriate change.